deepinfra/models/base/text_to_image.py

```diff
@@ -16,4 +16,4 @@ def generate(self, input):
         """
         body = {"input": input}
         response = self.client.post(body)
-        return TextToImageResponse(**response.json())
+        return response.json()
```

Comment:
Same here. You also don't have annotation in the generate method.

You can use template-like specialization in inheritance, so the generate method won't complain it has a different signature, if that is the problem.